mappings/net/minecraft/entity/mob/EntityCaveSpider.mapping

```diff
@@ -1,5 +1,7 @@
 CLASS abk net/minecraft/entity/mob/EntityCaveSpider
 	METHOD J getLootTableId ()Lmd;
 	METHOD a registerTagUpgraders (Lqv;)V
+	METHOD a (Lsv;Lun;)Lun;
```

Comment:
Naming a parameter without naming the function was outlined in the style guide as something not to do, although I'm not sure if it's current...